tests/robot-tests/tests/libs/utilities.py

```diff
@@ -343,3 +343,7 @@ def remove_substring_from_right_of_string(string, substring):
         return string[:-len(substring)]
     else:
         raise_assertion_error(f'String "{string}" doesn\'t end with substring "{substring}"')
+
+def user_clicks_element_if_exists(selector):
```

Comment:
I think we could just do this with Robot keywords fairly easily

Reply:
Discussed on Slack. Tried using `run keyword and ignore error`, but a screenshot is still getting created when the keyword fails, so going to stick with the python function.